src/Amazon.Lambda.Tools/Commands/UpdateFunctionConfigCommand.cs

```diff
@@ -623,6 +643,66 @@ private UpdateFunctionConfigurationRequest CreateConfigurationRequestIfDifferent
                 }
             }
 
+            var logFormat = this.GetStringValueOrDefault(this.LogFormat, LambdaDefinedCommandOptions.ARGUMENT_LOG_FORMAT, false);
+            if (!string.IsNullOrEmpty(logFormat))
+            {
+                if (request.LoggingConfig == null)
+                {
+                    request.LoggingConfig = new LoggingConfig();
+                }
+
+                if (!string.Equals(request.LoggingConfig.LogFormat, existingConfiguration.LoggingConfig?.LogFormat, StringComparison.Ordinal))
+                {
+                    request.LoggingConfig.LogFormat = logFormat;
+                    different = true;
+                }
+            }
+
+            var logApplicationLevel = this.GetStringValueOrDefault(this.LogApplicationLevel, LambdaDefinedCommandOptions.ARGUMENT_LOG_APPLICATION_LEVEL, false);
+            if (!string.IsNullOrEmpty(logApplicationLevel))
+            {
+                if (request.LoggingConfig == null)
+                {
+                    request.LoggingConfig = new LoggingConfig();
+                }
+
+                if (!string.Equals(request.LoggingConfig.ApplicationLogLevel, existingConfiguration.LoggingConfig?.ApplicationLogLevel, StringComparison.Ordinal))
+                {
+                    request.LoggingConfig.ApplicationLogLevel = logApplicationLevel;
+                    different = true;
+                }
+            }
+
+            var logSystemLevel = this.GetStringValueOrDefault(this.LogSystemLevel, LambdaDefinedCommandOptions.ARGUMENT_LOG_APPLICATION_LEVEL, false);
```

Comment:
Shouldn't you be checking "ARGUMENT_LOG_SYSTEM_LEVEL"?

Reply:
Fixed